src/Pages/Missions/Missions.jsx

```diff
@@ -0,0 +1,39 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Mission from '../../Components/Mission/Mission';
+import './Missions.css';
+import './Missions.scss';
+import { getMissions } from '../../Redux/Missions/mission';
+
+const MissionPage = () => {
+  const dispatch = useDispatch();
+
+  const missions = useSelector((state) => state.missions);
+  useEffect(() => {
+    if (missions.length) return;
+    dispatch(getMissions());
+  }, [dispatch, missions.length]);
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20you%20only%20dispatch%20those%20actions%20once%20and%20do%20not%20add%20data%20to%20store%20on%20every%20re%2Drender%20(i.e.%20when%20changing%20views%20/%20using%20navigation).), Please make sure that actions for getting missions dispatched once because when I load the missions page the actions dispatched twice.

![image](https://user-images.githubusercontent.com/68796232/208940530-78367eee-e0b6-4270-aa37-637c203a659e.png)

[Hint] See this [video](https://www.youtube.com/watch?v=Tay06Rk881Y) about how to use the redux toolkit for creating async thunk and creating a slice will help.

Reply:
Fixed.